internal/blockchain/common/common.go

```diff
@@ -334,7 +334,7 @@ func buildBatchPin(ctx context.Context, event *blockchain.Event, params *BatchPi
 func GetNamespaceFromSubName(subName string) string {
 	var parts = strings.Split(subName, "-")
 	// Subscription names post version 1.1 are in the format `ff-sub-<namespace>-<listener ID>`
-	if len(parts) != 4 {
+	if len(parts) < 4 {
```

Comment:
Is there a possibility that an old subscription could still hit this codepath if it contains a UUID?

Reply:
Yeaaa there is... 🫠

https://github.com/kaleido-io/firefly/commit/969d87e1cc68629a24a42acfa9392b1820461ef0#diff-b5bcf99952b7e15d54b1cb08438fb2337c46db9f1ba8d4bce463ef4a0dfb9168